website/src/docs/image-editor.md

```diff
@@ -121,6 +121,16 @@ uppy.on('file-editor:complete', (updatedFile) => {
 })
 ```
 
+### file-editor:cancel
+
+Emitted when `uninstall` is called.
```

Comment:
Shouldn't we also call this event when you are editing, but discard the changes?

Reply:
I'm trying to use this with uppy dashboard and it does not fire when changes are discarded. I might have understood it wrong but this event does not get triggered at all @Murderlon. 

This is what I'm trying to test with
```ts
    uppy.on("file-editor:cancel", (e) => {
      console.log("CANCEL")
    })
```
and hitting this button is when I'm expecting it to get called
<img width="369" alt="image" src="https://user-images.githubusercontent.com/48270786/186370798-f830b5d5-24d9-4426-bd2b-cabb3b461155.png">

Reply:
Are you on the latest version?

Reply:
Sorry for the late reply, I'm on [2.13.4](https://github.com/transloadit/uppy/releases/tag/uppy%402.13.4) @Murderlon

Reply:
Can you update to latest 2.x (or 3.x) and try again?

Reply:
2.3.3 is the latest which I'm on. Are you able to repro this when using the image editor with dashboard @Murderlon?

Reply:
This was shipped in [2.13.0](https://github.com/transloadit/uppy/releases/tag/uppy%402.13.0)

Reply:
Created a new issue: https://github.com/transloadit/uppy/issues/4045